client/components/domains/connect-domain-step/page-definitions.js

```diff
@@ -93,6 +95,87 @@ export const connectADomainStepsDefinition = {
 	},
 };
 
+export const connectASubdomainStepsDefinition = {
+	// Suggested flow
+	[ stepSlug.SUBDOMAIN_SUGGESTED_START ]: {
+		mode: modeType.SUGGESTED,
+		step: stepType.START,
+		component: ConnectDomainStepSuggestedStart,
+		next: stepSlug.SUBDOMAIN_SUGGESTED_LOGIN,
+	},
+	[ stepSlug.SUBDOMAIN_SUGGESTED_LOGIN ]: {
+		mode: modeType.SUGGESTED,
+		step: stepType.LOG_IN_TO_PROVIDER,
+		get name() {
+			return __( 'Log in to provider' );
+		},
+		component: ConnectDomainStepLogin,
+		next: stepSlug.SUBDOMAIN_SUGGESTED_UPDATE,
+		prev: stepSlug.SUBDOMAIN_SUGGESTED_START,
+	},
+	[ stepSlug.SUBDOMAIN_SUGGESTED_UPDATE ]: {
+		mode: modeType.SUGGESTED,
+		step: stepType.UPDATE_NS_RECORDS,
+		get name() {
+			return __( 'Update NS records' );
+		},
+		component: ConnectSubdomainStepSuggestedRecords,
+		prev: stepSlug.SUBDOMAIN_SUGGESTED_LOGIN,
+	},
+	[ stepSlug.SUBDOMAIN_SUGGESTED_CONNECTED ]: {
+		mode: modeType.SUGGESTED,
+		step: stepType.CONNECTED,
+		component: ConnectDomainStepDone, // TODO: change
+		prev: stepSlug.SUBDOMAIN_SUGGESTED_UPDATE,
+	},
+	[ stepSlug.SUBDOMAIN_SUGGESTED_VERIFYING ]: {
+		mode: modeType.SUGGESTED,
+		step: stepType.VERIFYING,
+		component: ConnectDomainStepDone,
+		prev: stepSlug.SUBDOMAIN_SUGGESTED_UPDATE,
+	},
+
+	// Advanced flow
+	[ stepSlug.SUBDOMAIN_ADVANCED_START ]: {
+		mode: modeType.ADVANCED,
+		step: stepType.START,
+		component: ConnectDomainStepAdvancedStart,
+		next: stepSlug.SUBDOMAIN_ADVANCED_LOGIN,
+		prev: stepSlug.SUBDOMAIN_SUGGESTED_START,
+	},
+	[ stepSlug.SUBDOMAIN_ADVANCED_LOGIN ]: {
+		mode: modeType.ADVANCED,
+		step: stepType.LOG_IN_TO_PROVIDER,
+		get name() {
+			return __( 'Log in to provider' );
+		},
+		component: ConnectDomainStepLogin,
+		next: stepSlug.SUBDOMAIN_ADVANCED_UPDATE,
+		prev: stepSlug.SUBDOMAIN_ADVANCED_START,
+	},
+	[ stepSlug.SUBDOMAIN_ADVANCED_UPDATE ]: {
+		mode: modeType.ADVANCED,
+		step: stepType.UPDATE_CNAME_RECORDS,
+		get name() {
+			return __( 'Update CNAME records' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Update root A records & CNAME record' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/subdomain-connection-instructions)